src/org/gk/model/InstanceUtilities.java

```diff
@@ -1394,4 +1396,62 @@ else if (instance.getSchemClass().isa(ReactomeJavaConstants.Pathway))
 	//public static String encodeLineSeparators(String text) {
 	//    return text.replaceAll(FileAdaptor.LINE_END + "", "<br>");
 	//}
+
+     /**
+      *
+      * @param id
+      * @return
+      */
+     public static boolean isDrug(Long id) {
+         if (id == null)
+             return false;
+
+         XMLFileAdaptor fileAdaptor = PersistenceManager.getManager().getActiveFileAdaptor();
+         return isDrug(fileAdaptor.fetchInstance(id));
+     }
+
+     /**
+      * Determines if an instance is a drug or contains a drug (in the case of EntitySets and Complexes).
+      *
+      * @param instance
+      * @throws Exception
+      * @returns boolean
+      */
+     public static boolean isDrug(Object instance) {
```

Comment:
Why this is Object, not GKInstance?

Reply:
Fixed in [ad58739](https://github.com/reactome/CuratorTool/pull/26/commits/3822611dfb0486ef3eb45d492b83cc810ad58739).